jcenter/bintray.gradle

```diff
@@ -106,3 +107,17 @@ bintray {
     }
   }
 }
+
```

Comment:
Please add the following lines:

publishToMavenLocal.dependsOn assemble
publishToMavenLocal.dependsOn sourcesJar

Reply:
Added.